straxen/plugins/peaklet_processing.py

```diff
@@ -543,13 +543,21 @@ def compute(self, records_he, start, end):
 
 @export
 @strax.takes_config(
-    strax.Option('s1_max_rise_time', default=110, infer_type=False,
-                 help="Maximum S1 rise time for < 100 PE [ns]"),
-    strax.Option('s1_max_rise_time_post100', default=200, infer_type=False,
+    strax.Option('s1_risetime_area_parameters', default=(50, 80, 12), type=(list, tuple),
+                 help="norm, const, tau in the empirical boundary in the risetime-area plot"),
+    strax.Option('s1_risetime_aft_parameters', default=(-1, 2.6), type=(list, tuple),
+                 help=("Slope and offset in exponential of emperical boundary in the rise time-AFT "
+                      "plot. Specified as (slope, offset)")),
+    strax.Option('s1_flatten_threshold_aft', default=(0.7, 100), type=(tuple, list),
```

Comment:
@yuema137 sorry Yue, but it's not clear to me which values (100ns or 80ns) should use be used here. I'm asking because in[ your note](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:yue:classification_manual_boundary) it is defined as 80ns, but here it is defined as 100ns. Which one is correct?

Reply:
OK, now I found [your new note](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:yue:classification_manual_boundary_update_1), indicating that 100 ns should be the correct one, but here `s1_flatten_threshold_aft` value of 0.6 is suggested to use. Which one (0.6 or 0.7) is correct?

Reply:
0.6 is correct and I have fixed that. 
As we discussed earlier, it is more safe to use a loose boundary for now. So I changed it from (0.7, 80) to (0.6, 100). Sorry for the mistake.